src/converters.ts

```diff
@@ -268,75 +307,98 @@ export function transformCredentialInput(
   return result as JwtCredentialPayload
 }
 
-function normalizeJwtPresentationPayload(input: DeepPartial<JwtPresentationPayload>): W3CPresentation {
+function normalizeJwtPresentationPayload(
+  input: DeepPartial<JwtPresentationPayload>,
+  removeOriginalFields: boolean = false
```

Comment:
as mentioned above I think we do not need this property in the JWT->VC case

Reply:
see #64